client/optimise.py

```diff
@@ -25,7 +25,7 @@ def get_job_properties(self, world_id: str, job_reference: str) -> Dict[Any, Any
             f"worlds/{world_id}/jobs/{job_reference}/additionalProperties"
         ).json()
 
-    def get_worlds(self) -> List[Any]:
+    def get_worlds(self) -> List[Dict[str, str]]:
```

Comment:
Technically it's not a `Dict[str, str]`, looking at the tests it could be a `Dict[str, Union[str, Dict[str, str]]` or we create a type that actually matches the model.